.github/workflows/seldon_kind_test.yaml

```diff
@@ -0,0 +1,32 @@
+name: Build & Apply Seldon manifests in KinD
+on:
+  pull_request:
+    paths:
+      - contrib/seldon/**
```

Comment:
I think this should be okay for now, but @kimwnasptd we should think about other ways to trigger `/contrib` tests in the future and not just trigger based on changes to the `contrib/seldon` directory.

Reply:
Was thinking the same thing! We could probably use the `workflow_dispatch` event as well in these workflows, to be able to manually trigger them https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch

Let's play around with this more once we have the components update their manifests and have some basic tests in place.